MH4U Dex/AppDelegate.m

```diff
@@ -0,0 +1,43 @@
+//
+//  AppDelegate.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/5/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+#import "HomeViewController.h"
+#import "CoreDataController.h"
+
+@interface AppDelegate ()
+
+@property (nonatomic, strong) CoreDataController *coreDataController;
+
+@end
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    NSLog(@"Loading Data");
+    self.coreDataController = [[CoreDataController alloc] init];
+    [self.coreDataController loadMonsterData];
+    [self.coreDataController loadMonsterDropData];
+    [self.coreDataController loadRegionData];
```

Comment:
Changed.  This singleton pattern is really useful, now I don't have to worry about passing the managedObjectContext everywhere!